assets/locales/service.cy.toml

```diff
@@ -206,3 +206,15 @@ one = "See version history"
 [CensusDatasetLandingPageMetadataLabel]
 description = "Census DLP metadata label - datestamp and dataset ID"
 one = "Dataset ID and release date information"
+
+[GetDataLeadText]
+description = "Get the data component lead text"
+one = "Download the data in the following formats"
+
+[GetDataIncludesSupportingInfo]
```

Comment:
I think this and the supporting info localisations are generic enough that they won't just be used in a "Get Data" context; I'd consider removing the GetData prefix from the keys